tutorial/sample-problems/DiffCalc/AnswerWithUnits.pg

```diff
@@ -12,6 +12,8 @@
 ## MO(1)
 ## KEYWORDS('differential calculus', 'units')
 
+# Updated 2025.02 to use the contextUnits.pl macro.
+
```

Comment:
Do we want to get into the habit of adding changelog comments to some of these examples? I don't initially see the point to this.

Reply:
I also did not like that this comment was added.  It should be removed.

Reply:
```suggestion
```